README.md

```diff
@@ -96,19 +96,12 @@ Click the button below to start deployment.
    **Note:** This value defaults to `insight-operational-logs`. This Event Hub is created automatically by Azure when a subscription [Log Profile is integrated with Event Hub through the Azure Monitor service](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/stream-monitoring-data-event-hubs#azure-subscription-monitoring-data).
    Follow this guide to [Stream the Azure Activity Log to Event Hubs](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/activity-logs-stream-event-hubs).
 
+   - **Event Hub Filter** - Two types of filtering are supported, Event Hub Filter Regex and Event Hub Filter Json. For more details click [here](#event-hub-filtering)
```

Comment:
please keep it as two parameters as it was before.
```
   - **Event Hub Filter Json** - Type the filter in JSON format.  **For example:** {"resultType":"Success"}

   - **Event Hub Filter Regex** - Type the filter in REGEX format. **For example:** `\/*.Policy or "Policy"`. For more details click [here](#event-hub-filtering)

   **Note:** For `Event Hub Filter Json` and `Event Hub Filter Regex`, only messages which contain the specified property will be collected. If both the filter values are provided then logs will be collected based on both the values.
```

Reply:
Done